build/ReleaseNotesBuilder/ReleaseNotesModel.cs

```diff
@@ -0,0 +1,225 @@
+﻿// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Text;
+
+namespace ReleaseNotesBuilder
+{
+    internal class ReleaseNotesModel
+    {
+        private const string FrontMatterWrapper = "---";
+        private const string FrontMatterSubject = "subject";
+        private const string FrontMatterReleaseDate = "releaseDate";
+        private const string FrontMatterVersion = "version";
+        private const string FrontMatterDownloadLink = "downloadLink";
+        private const string FrontMatterFeatures = "features";
+        private const string FrontMatterBugs = "bugs";
+        private const string FrontMatterSecurity = "security";
+
+        private readonly string? _subject;
+        private readonly string? _releaseDate;
+        private string? _releaseVersion;
+        private readonly string? _downloadLink;
+        private readonly List<string> _frontFeatures;
+        private readonly List<string> _frontBugs;
+        private readonly List<string> _frontSecurity;
+
+        private readonly string? _preamble;
+        private readonly List<string> _bodyNewFeatures;
+        private readonly List<string> _bodyFixes;
+        private readonly List<string> _bodySecurity;
+        private readonly List<string> _bodyNotice;
+        private readonly string _checksums;
+        private readonly string? _epilogue;
+
+        public ReleaseNotesModel(PersistentData persistentData, string checksums)
+        {
+            _frontFeatures = new List<string>();
+            _frontBugs = new List<string>();
+            _frontSecurity = new List<string>();
+            _bodyNewFeatures = new List<string>();
+            _bodyFixes = new List<string>();
+            _bodySecurity = new List<string>();
+            _bodyNotice = new List<string>();
+
+            _subject = persistentData.Subject;
+            _releaseDate = DateTime.Now.ToString("yyyy-MM-dd");
+            _downloadLink = persistentData.DownloadLink;
+            _preamble = persistentData.Preamble;
+            _epilogue = persistentData.Epilogue;
+            _checksums = checksums;
+        }
+
+        public string Make()
```

Comment:
A more descriptive name for this method might make it more self-documenting. Maybe something like `GetDocumentContents`, `GetFormattedContents`, or `FormatForDocsSite`?

Reply:
Will do!